src/cmd/sanity_test/export/sharepoint.go

```diff
@@ -14,15 +20,65 @@ func CheckSharePointExport(
 	ac api.Client,
 	envs common.Envs,
 ) {
-	drive, err := ac.Sites().GetDefaultDrive(ctx, envs.SiteID)
-	if err != nil {
-		common.Fatal(ctx, "getting the drive:", err)
+	if envs.Category == "lists" {
```

Comment:
category checks should compare against `path` package consts (or their string values) for consistency.  Avoid hardcoded strings if possible.